samples/04_gis_analysts_data_scientists/vehicle_detection_and_tracking.ipynb

```diff
@@ -0,0 +1,1305 @@
+{
```

Comment:
<p>I assume this data_path points to the folder of results from export training data. If so, we should spell it out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>